src/uct/cuda/cuda_ipc/cuda_ipc_iface.c

```diff
@@ -95,9 +95,10 @@ static ucs_status_t uct_cuda_ipc_iface_query(uct_iface_h tl_iface,
                                           UCT_IFACE_FLAG_CONNECT_TO_IFACE       |
                                           UCT_IFACE_FLAG_PENDING                |
                                           UCT_IFACE_FLAG_GET_ZCOPY              |
-                                          UCT_IFACE_FLAG_PUT_ZCOPY              |
-                                          UCT_IFACE_FLAG_EVENT_SEND_COMP        |
-                                          UCT_IFACE_FLAG_EVENT_RECV;
+                                          UCT_IFACE_FLAG_PUT_ZCOPY;
+    iface_attr->cap.event_flags         = UCT_IFACE_FLAG_EVENT_SEND_COMP |
```

Comment:
What about ROCM and Ugni ? is it 0 ?

Reply:
neither RoCM IPC nor UGNI TLs support event notification mechanisms
yes, this is 0 for them